shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-sqlserver/src/main/antlr4/imports/sqlserver/DMLStatement.g4

```diff
@@ -73,11 +73,15 @@ multipleTableNames
     ;
 
 select 
-    : unionClause
+    : aggregationClause
+    ;
+
+aggregationClause
+    : selectClause ((UNION (ALL)? | EXCEPT | INTERSECT) selectClause)*
     ;
 
 unionClause
-    : selectClause (UNION (ALL | DISTINCT)? selectClause)*
+    : selectClause (UNION (ALL)? selectClause)*
     ;
```

Comment:
Can these two rules be combined? Since the reference in the subquery is unionClause. The subquery and SELECT should have the same semantics,right?

Reply:
Hi @jingshanglu , You are right, I consider merge `unionClause` to  `aggregationClause` next PR at beginning, 
now I will do it in advance.